CHANGELOG.md

```diff
@@ -17,7 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add optional parameter `deviceLabels: DeviceLabels | DeviceLabelTrigger` in `meetingManager.listAndSelectDevices()` to let builder indicate the type of devices they want to select.
-- Add several how-to docs in storybook.
+- Add optional parameter `logger` and `videoDownlinkBandwidthPolicy` in `ManagerConfig` to pass in the `ConsoleLogger` instance and `VideoDownlinkBandwidthPolicy` instance that you want to be used in the meeting session.
```

Comment:
Based on previous learning, it is recommended to say "object" over "instance". Had received one comment on an earlier PR of mine.
https://github.com/aws/amazon-chime-sdk-component-library-react/pull/554#discussion_r667229752

Reply:
Yes sir, good point, will address all.